webpack-main-prod.config.js

```diff
@@ -0,0 +1,10 @@
+const merge = require('webpack-merge');
+const commonConfig = require('./webpack-main.config.js');
+
+module.exports = merge(commonConfig, {
+    mode: 'production',
+    devtool: 'source-map',
```

Comment:
I guess there's not much loss in including source map. Code obfuscation is not my concern.